server/helpers/image-utils.ts

```diff
@@ -1,7 +1,7 @@
 import 'multer'
-import * as sharp from 'sharp'
 import { readFile, remove } from 'fs-extra'
 import { logger } from './logger'
+const Jimp = require('jimp')
```

Comment:
Try `import jimp from 'jimp'` instead

Reply:
It requires `esModuleInterop`, which we don't seem to support(?).